<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClassAttributesEditorUI</class>
 <widget class="QWidget" name="CClassAttributesEditorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Attributes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="GraphButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>23</height>
        </size>
       </property>
       <property name="statusTip">
        <string>Edit graph attributes</string>
       </property>
       <property name="text">
        <string>Graph</string>
       </property>
       <property name="icon">
        <iconset resource="commonui.qrc">
         <normaloff>:/Icons/Graph</normaloff>:/Icons/Graph</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="NodeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>23</height>
        </size>
       </property>
       <property name="statusTip">
        <string>Edit default node attributes</string>
       </property>
       <property name="text">
        <string>Node</string>
       </property>
       <property name="icon">
        <iconset resource="commonui.qrc">
         <normaloff>:/Icons/Node-Disc</normaloff>:/Icons/Node-Disc</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="EdgeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>23</height>
        </size>
       </property>
       <property name="statusTip">
        <string>Edit default edge attributes</string>
       </property>
       <property name="text">
        <string>Edge</string>
       </property>
       <property name="icon">
        <iconset resource="commonui.qrc">
         <normaloff>:/Icons/Edge-Undirected</normaloff>:/Icons/Edge-Undirected</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CPropertyEditorUIBase" name="Editor"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="AddButton">
       <property name="statusTip">
        <string>Add new custom attribute for the selected items</string>
       </property>
       <property name="text">
        <string>Add...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ChangeButton">
       <property name="statusTip">
        <string>Change selected attribute's id and type</string>
       </property>
       <property name="text">
        <string>Change...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemoveButton">
       <property name="statusTip">
        <string>Remove the attribute from the selected items</string>
       </property>
       <property name="text">
        <string>Remove...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPropertyEditorUIBase</class>
   <extends>QTreeWidget</extends>
   <header location="global">commonui/CPropertyEditorUIBase.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>GraphButton</tabstop>
  <tabstop>NodeButton</tabstop>
  <tabstop>EdgeButton</tabstop>
  <tabstop>Editor</tabstop>
  <tabstop>AddButton</tabstop>
  <tabstop>ChangeButton</tabstop>
  <tabstop>RemoveButton</tabstop>
 </tabstops>
 <resources>
  <include location="commonui.qrc"/>
 </resources>
 <connections/>
</ui>
